etc/audacity.profile

```diff
@@ -18,8 +16,9 @@ include /etc/firejail/whitelist-var-common.inc
 
 apparmor
 caps.drop all
-#net none
+net none
 no3d
+# nodbus
```

Comment:
Should we uncomment this since /run/user/*/bus wasn't commented?

Reply:
It also makes settings immutable, but only in Fedora Gnome. I think @SkewedZeppelin observed this first. I can add a comment there to make it clearer.